src/components/FeatureAnnouncementBanner/index.jsx

```diff
@@ -37,10 +39,10 @@ const FeatureAnnouncementBanner = ({ enterpriseSlug }) => {
   }, [enterpriseSlug]);
 
   return (
-    isRead || !enterpriseNotificationBanner.text ? null : (
+    isRead || !enterpriseNotificationBanner.text || !enterpriseNotificationBanner.title ? null : (
```

Comment:
Can you please explain why do we need to add a separate condition for `title` here, wouldn't `!enterpriseNotificationBanner.text` enough?

Reply:
Just to apply the same validations of the backend. It will work fine without this check but I added it to handle issues if something changed in the future at the backend then we should have the same validation on the frontend.